openassessment/workflow/models.py

```diff
@@ -608,27 +612,35 @@ def is_cancelled(self):
         """
         return self.cancellations.exists()
 
+    @property
+    def identifying_uuid(self):
+        """
+        Returns the primary identifying uuid for this workflow.
+        Can be overriden by child classes
+        """
+        return self.submission_uuid
+
 
 class TeamAssessmentWorkflow(AssessmentWorkflow):
     """
     Extends AssessmentWorkflow to support team based assessments.
     """
     # Only staff assessments are supported for teams
-    STEPS = ['staff']
-    REQUIREMENTS = {"staff": {"required": True}}
+    TEAM_STAFF_STEP_NAME = 'teams'
```

Comment:
what do you mean?

Reply:
I think in other places (e.g. submissions) we have just left the step just called "staff" (though, in submissions we have a separate model so we can differentiate). Not sure how all the pieces integrate so I was wondering if having a different step name would cause any problems.

Reply:
Or if we need to update pieces downstream of the workflow to understand the "teams" step name instead of "staff"

Reply:
I'm pretty sure that this is only used for the Step model to grab the api/teams.py file. I don't believe any additional changes will need to be made (although i guess now that I say this, i'll have to take responsibility and be the one who has to fix them if they pop up xD)